### PR TITLE
[1LP][RFR] test_clear_datastore_filter_results fix

### PR DESCRIPTION
**Pull request by izapolsk**
{{pytest: cfme/tests/infrastructure/test_set_default_filter.py::test_clear_datastore_filter_results}}

### CODE REVIEW

**Comment by izapolsk** [comment]
My change fixed the issue.
However,  PRT fails because this test has to be redesigned. I think this has to be done by test owner.

File: cfme/tests/infrastructure/test_set_default_filter.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def test_clear_datastore_filter_results(appliance):
     """ Test for clearing filter results for datastores."""
     dc = DatastoreCollection(appliance)
     view = navigate_to(dc, 'All')
-    view.sidebar.datastores.tree.click_path('Datastores', 'All Datastores', 'Global Filters',
+    view.sidebar.datastores.tree.click_path('All Datastores', 'Global Filters',
```
Comment (quarckster) [inline cfme/tests/infrastructure/test_set_default_filter.py]: Is it correct for all versions of CFME?

Comment (izapolsk) [reply cfme/tests/infrastructure/test_set_default_filter.py]: yes, this is bug created during web_ui->widgetastic conversion

**Comment by mshriver** [comment]
@izapolsk You're correct, test makes it past this call now but still fails. FA owner should revise test.

**Pull request merged**